packages/fields/src/types/CloudinaryImage/ImageBlock.js

```diff
@@ -2,37 +2,24 @@ import pluralize from 'pluralize';
 import { Block } from '../../Block';
 import RelationshipFieldType from '../Relationship';
 import CloudinaryImage from './';
-import Text from '../Text';
-import { importView } from '@keystone-alpha/build-field-types';
 
-export class CloudinaryBlock extends Block {
-  static get type() {
-    return 'cloudinaryImage';
-  }
-
-  static get isComplexDataType() {
-    return true;
-  }
-
-  static get viewPath() {
-    return importView('../Content/views/editor/blocks/image-container');
-  }
-
-  constructor({ adapter }, { fromList, createAuxList, getListByKey, listConfig }) {
+export class ImageBlock extends Block {
+  constructor({ adapter }, { type, fromList, createAuxList, getListByKey, listConfig }) {
     super();
 
     this.fromList = fromList;
+    this.type = type;
+
+    const auxListKey = `_Block_${fromList}_${this.type}`;
 
-    const auxListKey = `_Block_${fromList}_${CloudinaryBlock.type}`;
+    // Ensure the list is only instantiated once per server instance.
     let auxList = getListByKey(auxListKey);
 
     if (!auxList) {
       auxList = createAuxList(auxListKey, {
         fields: {
           // We perform the requires here to avoid circular dependencies
           image: { type: CloudinaryImage, isRequired: true, adapter },
-          from: { type: RelationshipFieldType, isRequired: true, ref: fromList },
-          field: { type: Text, isRequired: true },
```

Comment:
A touch of cleanup here too - these fields came in an earlier PR and make no sense yet (they may come back in a future PR)